frameworks/person-escort-record/README.md

```diff
@@ -97,6 +105,13 @@ Question files should live in [`questions/`](./questions) and be stored in a fla
 - `validations` - list of [validations](#question-validation) that need to be applied to this question
   - `type` **(required)** - the validation error key
   - `message` - custom text that will be displayed in the form
+  - `value` - further data required for some [validation types](#question-validation).
+
+#### Multiple type questions
```

Comment:
What about using `#### Questions types` as the heading, similar to the `Question validations` heading below? And it allows us to use this section to describe any types that might need more explanation.